app/jobs/application_job.rb

```diff
@@ -10,6 +10,10 @@ class ApplicationJob < ActiveJob::Base
   rescue_from(StandardError) do |e|
     NewRelic::Agent.notice_error(e)
   ensure
-    raise e
+    if e.is_a? ActiveJob::DeserializationError
+      Rails.logger.warn(e.message)
```

Comment:
Was hitting some errors with callbacks coming in for deleted records.  Seems safe enough to just log these as a warning and move on?